server.js

```diff
@@ -1,12 +1,53 @@
-const express = require('express');
-const path = require('path');
+const express = require('express')
+const path = require('path')
+const axios = require('axios')
+const app = express()
+const https = require('https')
+const port = process.env.PORT || 5001
+const isProduction = process.env.NODE_ENV === 'production'
 
-const app = express();
-const port = process.env.PORT || 5001;
+async function requestHostFromKubernetes() {
+  const response = await axios({
+    url: 'https://kubernetes.default/.well-known/oauth-authorization-server',
+    httpsAgent: new https.Agent({ rejectUnauthorized: false })
```

Comment:
This ignores SSL errors. I was going to use `isProduction` to determine this, i.e in production we'd expect a valid SSL cert (`rejectUnauthorized: true`) but in dev we can ignore the cert errors (`rejectUnauthorized: false`). That doesn't work though since `isProduction` _has_ to be true to expose the assets both during development and production using `express.static`.

Could possibly add a new env variable to manage this?

Reply:
If you need to add a new env variable to support this you can go ahead and add it to the .env files.